packages/lesswrong/components/tagging/TagMergePage.tsx

```diff
@@ -0,0 +1,236 @@
+      <Typography className={classes.title} variant="title">
+        {taggingNameCapitalSetting.get()} merging tool
+      </Typography>
+      <div className={classes.section}>
+        <div className={classes.selectTagRow}>
+          <div className={classes.tagSection}>
+            <div className={classes.tagSearch}>
+              <TagsSearchAutoComplete clickAction={setSourceTagId} placeholder={`Source ${taggingNameSetting.get()}`} />
+            </div>
+            {sourceTag ? (
+              <div className={classes.tagSummary}>
+                <Link to={tagGetUrl(sourceTag)} target="_blank" rel="noopener noreferrer">
+                  {sourceTag.name}
+                </Link>
+                <div>Slug: {sourceTag.slug}</div>
+                <div>_id: {sourceTag._id}</div>
+                <div>Post count: {sourceTag.postCount}</div>
+                <div>Subtag count: {sourceTag.subTags?.length ?? 0}</div>
+              </div>
+            ) : (
+              sourceTagId && sourceTagLoading && <Loading />
+            )}
+          </div>
+          <div className={classes.tagSection}>
+            <div className={classes.tagSearch}>
+              <TagsSearchAutoComplete clickAction={setTargetTagId} placeholder={`Target ${taggingNameSetting.get()}`} />
+            </div>
+            {targetTag ? (
+              <div className={classes.tagSummary}>
+                <Link to={tagGetUrl(targetTag)} target="_blank" rel="noopener noreferrer">
+                  {targetTag.name}
+                </Link>
+                <div>Slug: {targetTag.slug}</div>
+                <div>_id: {targetTag._id}</div>
+                <div>Post count: {targetTag.postCount}</div>
+                <div>Subtag count: {targetTag.subTags?.length ?? 0}</div>
+              </div>
+            ) : (
+              targetTagId && targetTagLoading && <Loading />
+            )}
+          </div>
+        </div>
+        <div className={classes.checkboxRow}>
+          <LWTooltip title="Disabling this not implemented">
+            <Checkbox
+              className={classes.checkbox}
+              checked={transferPosts}
+              onChange={(_, checked) => setTransferPosts(checked)}
+              disabled={true}
+            />
+          </LWTooltip>
+          <Typography className={classes.inline} variant="body2" component="label">
+            Transfer posts
+          </Typography>
+        </div>
+        <div className={classes.checkboxRow}>
+          <LWTooltip title="Disabling this not implemented">
+            <Checkbox
+              className={classes.checkbox}
+              checked={removePosts}
+              onChange={(_, checked) => setRemovePosts(checked)}
+              disabled={true}
+            />
+          </LWTooltip>
+          <Typography className={classes.inline} variant="body2" component="label">
+            {`Remove posts from source ${taggingNameSetting.get()}`}
+          </Typography>
+        </div>
```

Comment:
On why these are here but disabled: I think it would be nice to make these independent options, but when both of them are checked the backend is a bit more straightforward because we can just change the `tagId` on the existing TagRel rather than creating a new one. This is all that is needed to merge the specific tags I have, so I decided to not implement these separately.

Still, I think having them here greyed out is a good way to explain what the tool does, and nudge future devs to implement them if needed